otx/algorithms/segmentation/adapters/mmseg/models/necks/selfsl_mlp.py

```diff
@@ -0,0 +1,105 @@
+"""Multi-layer Perceptron (MLP) for Self-supervised learning methods.
+
+This MLP consists of fc (conv) - norm - relu - fc (conv).
+"""
+
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+# pylint: disable=dangerous-default-value
+from typing import Any, Dict
+
+import torch
+from mmcv.cnn import build_norm_layer, kaiming_init, normal_init
+from mmseg.models.builder import NECKS  # pylint: disable=no-name-in-module
+from torch import nn
+
+
+@NECKS.register_module()
+class SelfSLMLP(nn.Module):
```

Comment:
Is this neck class exactly same with the one in classification? Actually I don't really think that reusing is always better than copying. And as far as I know, it is plan to support the all the model replacement not only for backbone in the futher. I think it's okay to leave it as is. But if needed, I suggest to import the registry function dynamically checking the task type argument when this neck class is called initially.

Reply:
Exactly same. I'd like to know the reason why you don't think that reusing is always better than copying.
And following the plan you said, I'll update this module later if it is necessary to combine them.